addons/portal_rating/static/src/xml/portal_chatter.xml

```diff
@@ -72,7 +72,7 @@
 
     <t t-name="portal_rating.chatter_rating_publisher_comment">
         <div class="d-flex o_portal_chatter_message">
```

Comment:
We tend to avoid using custom css when it's possible to use bootstrap classes.
Instead of your `margin-right: 1rem` you can simply set a `gap-2` on this div :slightly_smiling_face: